lib/menubar/utils.lua

```diff
@@ -290,14 +290,14 @@ function utils.parse_dir(dir_path, callback)
         local query = gio.FILE_ATTRIBUTE_STANDARD_NAME .. "," .. gio.FILE_ATTRIBUTE_STANDARD_TYPE
         local enum, err = file:async_enumerate_children(query, gio.FileQueryInfoFlags.NONE)
         if not enum then
-            gdebug.print_error(err)
+            gdebug.print_warning(dir_path .. ": " .. tostring(err))
             return
         end
         local files_per_call = 100 -- Actual value is not that important
         while true do
             local list, enum_err = enum:async_next_files(files_per_call)
             if enum_err then
-                gdebug.print_error(enum_err)
+                gdebug.print_warning(dir_path .. ": " .. tostring(enum_err))
```

Comment:
Could this use `local path = file:get_path() or file:get_uri()` instead of just `dir_path`? `dir_path` is just the directory that we started with, but some subdirectory could be throwing some errors, I think (although I'm not really sure what kind of errors... `chmod 0` would already be caught earlier...?)